pkgs/by-name/li/libinput-three-finger-drag/package.nix

```
@@ -0,0 +1,38 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  libinput,
+  xdotool,
+  makeWrapper,
+}:
+
+rustPlatform.buildRustPackage {
+  pname = "libinput-three-finger-drag";
+  version = "0.1";
+
+  src = fetchFromGitHub {
+    owner = "marsqing";
+    repo = "libinput-three-finger-drag";
+    rev = "master";
```

Use a pinned release. 
```suggestion
    tag = version;
```

---

The version tag in the original repository is unfortunately broken. What I can do, is to use commit hash instead of master - does it make more sense here?

---

Yup — do `rev = "<commit SHA>"` and rename the version to `0.1-unstable-YYYY-MM-DD`, where YYYY-MM-DD is the date of the commit

---

Done :+1: